package.json

```diff
@@ -506,12 +506,12 @@
                 },
                 {
                     "command": "vscode-docker.registries.copyRemoteFullTag",
-                    "when": "view == dockerRegistries && viewItem =~ /(dockerHubTag|registryV2Tag|azureContainerTag)/i",
+                    "when": "view == dockerRegistries && viewItem =~ /(dockerHubTag|genericRegistryV2Tag|azureContainerTag|githubRegistryTag)/i",
```

Comment:
should we change this to (dockerHubTag|registryV2Tag)?

Reply:
the advantage would be any RDP class that extends v2 registry would be able to have the command show up

Reply:
Yes, I think that's a good idea.